Frontend/src/redux/thunks/userThunks.js

```diff
@@ -0,0 +1,84 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+const axios = require('axios').default;
+
+import {addUser} from '../actions/userActions';
+
+export const loginUser = ({user, navigation}) => {
+  return async () => {
+    try {
+      const response = await axios.post(
+        `http://192.168.100.2:3000/api/user/login`,
+        user,
+      );
+
+      await AsyncStorage.setItem('accessToken', response.data.accessToken);
+      navigation.replace('TabNavigatorRoutes');
+    } catch (error) {
+      alert(error.response.data.message);
+    }
+  };
+};
+
+export const registerUser = ({user}) => {
+  return async () => {
+    try {
+      await axios.post(`http://192.168.100.2:3000/api/user/register`, user);
+
+      alert('Registration successful.');
+    } catch (error) {
+      alert(error.response.data.message);
+    }
+  };
+};
+
+export const saveUser = () => {
+  return async (dispatch) => {
+    const token = await AsyncStorage.getItem('accessToken');
+
+    try {
+      let user = await axios.get('http://192.168.100.2:3000/api/user', {
+        headers: {
+          authorization: token,
+        },
+      });
+
+      if (user.data.user) {
+        delete user.data.user.__v;
+        delete user.data.user.password;
```

Comment:
The password shouldn't be contained in the response.

Reply:
It wasn't contained in the response, removed unnecessary code.